Ahorn/entities/retractSpinner.jl

```diff
@@ -0,0 +1,22 @@
+module SpringCollab2020RetractSpinner
+
+using ..Ahorn, Maple
+
+@mapdef Entity "SpringCollab2020/RetractSpinner" RetractSpinner(x::Integer, y::Integer, attachToSolid::Bool=false)
+
+const placements = Ahorn.PlacementDict(
+    "Retract Spinner (Spring Collab 2020)" => Ahorn.EntityPlacement(
+        RetractSpinner
+    )
+)
+
+sprite = "danger/SpringCollab2020/retractspinner/urchin_harm00.png"
+
+function Ahorn.selection(entity::RetractSpinner)
+    x, y = Ahorn.position(entity)
+    return Ahorn.Rectangle(x - 8, y - 8, 16, 16)
```

Comment:
Any reasons this doesn't get the sprite size from image?

Reply:
Spinners are 2 tiles wide (16x16) but their textures are 24x24. This makes a weird overlap when you select a bunch of them placed next to each other.